capitals.rb

```diff
@@ -151,3 +151,80 @@
     name: "Wyoming",
     capital: "Cheyenne"
 }]
+
+states.shuffle!
+
+# def play
+  puts "++++++++++++++++++++++++++++++++++++++++++++++"
+  puts "Are ya ready kids??!!"
+  puts "Yes/No"
+  puts "++++++++++++++++++++++++++++++++++++++++++++++"
+
+  user_input_start = gets.chomp.to_s.upcase
+
+  if user_input_start == "YES"
+
+    puts "Lets Go!"
+
+    def play_game(arr1)
+
+      index = 0
```

Comment:
Using enumerables over loops allow us to avoid having to reference an index like this. Think about how you could `.each` in place of your `while` loop.